### PR TITLE
Lesson 1.2: React DOM And Components

### PR DESCRIPTION
**Pull request by Nata-Kalina**
Lesson 1.2: React DOM And Components:
1. Created a new file TodoList.jsx;
2. Moved list JSX from App.jsx to TodoList.jsx;
3. Moved todoList array from App.jsx to TodoList.jsx;
4. Created a new file AddTodoForm.jsx;
5. Wrote JSX for "Add Todo" form.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,26 +1,7 @@
...
-  {
-    id: '4',
-    title: 'Start learning Okta',
-  },
-];
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
 
 function App() {
   const [count, setCount] = useState(0);
```
Comment (gsvidal) [inline src/App.jsx:7]: You can remove the coun state, you won't need it.

File: src/AddTodoForm.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const AddTodoForm = () => {
+  return (
+    <>
+      <form>
+        <label htmlFor="todoTitle">Title: </label>
+        <input id="todoTitle" type="text" name="title" />
```
Comment (gsvidal) [inline src/AddTodoForm.jsx:8]: Good use of semantic html and accessibility when using the htmlFor attribute.

**Review by gsvidal — APPROVED** [review]
Nice work, everything looks good.

File: src/AddTodoForm.jsx (hunk 2 above)
Comment (Nata-Kalina) [reply src/AddTodoForm.jsx:8]: @gsvidal Thank you!

**Pull request merged**